src/components/Card.tsx

```diff
@@ -114,15 +115,17 @@ export default function ({
   nospace,
 }: ChildrenProp & CardProps) {
   return (
-    <div
-      className={cardContainer(shadow, color, onlyWrap, thin, small, nospace)}
-    >
-      {children}
-      {!!spinner && (
-        <div className={spinnerBox}>
-          <ArcText text={spinner} />
-        </div>
-      )}
-    </div>
+    <CardContext.Provider value={{ cardColor: color }}>
```

Comment:
Maybe we can use import statement like:
```TS
import {Provider} from 'components/CardContext'
```
?

Reply:
This export won't work :(